lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +12,12 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+    punc = ''''1234567890!"#$%&'()’*º+,-./:;<=>?@[\\]^_`{|}~'''
+    text = text.replace(' ', '')
```

Comment:
in one line

Reply:
Why do not you add this symbols to punctuation?

Reply:
fixed???????